### PR TITLE
Use gs/b/foo path on local disk

### PR DESCRIPTION
**Pull request by jbingham**
Also, removed a tricky test case that's very brittle and no one uses, so not worth continuing to keep alive.

### CODE REVIEW

**Comment by jbingham** [comment]
Note to reviewers: made local file paths friendlier, and added builder methods for folder copying.

File: src/main/java/com/google/cloud/genomics/dockerflow/task/TaskBuilder.java [hunk 1]
```diff
@@ -163,38 +163,61 @@ public TaskBuilder inputArray(String name, String delimiter, String[] values) {
...
+  public TaskBuilder inputFolder(String name, String defaultGcsPath, String localPath) {
+	return inputPath(name, defaultGcsPath, localPath, true);
+  }
+ 
+  /**
+   * Declare and assign an input parameter for a path in GCS, explicitly providing a relative
+   * local path. The file will be accessible in the Docker container at the absolute local path
+   * given by ${name}. The path can be either a file or a folder.
+   */
+  private TaskBuilder inputPath(String name, String defaultGcsPath, String localPath, boolean isFolder) {
```
Comment (mbookman) [inline src/main/java/com/google/cloud/genomics/dockerflow/task/TaskBuilder.java]: In a few similar contexts, we've discussed that "localPath" is unclear.
We've called the GCS path the "remotePath" or "remoteURI" and the localPath the "dockerPath".

File: src/test/java/com/google/cloud/genomics/dockerflow/examples/MultiLinearGraph.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ public static Task taskTwo() throws IOException {
         .input("message", "hello")
         .outputFile("outputFile")
         .docker("ubuntu")
-        .script("cp ${inputFile} ${outputFile} ; echo ${message} >> ${outputFile}")
+        .script("mkdir -p $(dirname ${outputFile}); cp ${inputFile} ${outputFile} ; echo ${message} >> ${outputFile}")
```
Comment (mbookman) [inline src/test/java/com/google/cloud/genomics/dockerflow/examples/MultiLinearGraph.java]: Can the dockerflow infrastructure create the output path?

File: src/main/java/com/google/cloud/genomics/dockerflow/task/TaskBuilder.java (hunk 1 above)
Comment (jbingham) [inline src/main/java/com/google/cloud/genomics/dockerflow/task/TaskBuilder.java]: I'm copying the names in the Pipelines API directly, and in the Dockerflow yaml file. 
If/when we update in Pipelines API, that seems like the time to push through the updates everywhere.

File: src/test/java/com/google/cloud/genomics/dockerflow/examples/MultiLinearGraph.java (hunk 2 above)
Comment (jbingham) [inline src/test/java/com/google/cloud/genomics/dockerflow/examples/MultiLinearGraph.java]: Yes, they can and should. Updating now.

**Pull request merged**